cms/envs/common.py

```diff
@@ -1981,6 +1981,17 @@
     }
 ]
 
+LIBRARY_BLOCK_TYPES = [
+    {
+        'component': 'library_sourced',
+        'boilerplate_name': None
```

Comment:
Does this `library_sourced` component not have a boilerplate file because it is a stand-in for other content?

Reply:
`boilerplate_name` is an optional YAML template. I just specified as `None` to omit. like in [line 1973](https://github.com/openedx/edx-platform/pull/30803/files#diff-a16fd1c6cf75bdebdf97a2633b533d39e43975c6e98af58787ed6af1282eca71R1973) (for ADVANCED_PROBLEM_TYPES)